src/algokit/cli/generate.py

```diff
@@ -111,19 +112,18 @@ def generate_client(app_spec: str, output: str, language: str | None) -> None:
                 "-a",
                 app_spec,
                 "-o",
-                output_path,
+                output,
             ],
-            check=True,
-            stdout=subprocess.DEVNULL,
-            stderr=subprocess.DEVNULL,
+            bad_return_code_error_message="npx failed, please check your npm",
         )
         logger.info(
-            f"Generating TypeScript client code for application specified in {app_spec} and writing to {output_path}"
+            f"Generating TypeScript client code for application specified in "
+            f"{app_spec_path} and writing to {output_path}"
         )
     elif language.lower() == "python":
-        generate_clients(app_spec=pathlib.Path(app_spec), output=pathlib.Path(output))
+        generate_clients(app_spec=app_spec_path, output=output_path)
 
-        # algokit_client_generator.generate_client(pathlib.Path(app_spec), output_path)
-        # logger.info(
-        #     f"Generating Python client code for application specified in {app_spec} and writing to {output_path}"
-        # )
+        algokit_client_generator.generate_client(app_spec_path, output_path)
```

Comment:
You should be able to remove these two lines now, as @inaie-makerx has added generate_clients which will call it already